sdk/core/azure-core/src/main/java/com/azure/core/util/serializer/AvroSerializerProxy.java

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.util.serializer;
+
+import java.util.Iterator;
+import java.util.ServiceLoader;
+
+/**
+ * Service Provider Interface (SPI) proxy for {@link AvroSerializerProvider}.
+ */
+public final class AvroSerializerProxy {
```

Comment:
Instead of using `Proxy`, can we use `AvroSerializerProviders` similar to `HttpClientProviders`? Just to avoid Proxy being mistaken for the networking term.